### PR TITLE
Add perf build.py and util

### PR DESCRIPTION
**Pull request by kalman**

### CODE REVIEW

**Comment by kalman** [comment]
Hopefully this is the last one.

File: js/perf/viewer/build.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
```
Comment (arv) [inline js/perf/viewer/build.py]: Who is calling this file?

**Comment by arv** [comment]
BTY

File: js/perf/viewer/build.py (hunk 1 above)
Comment (kalman) [inline js/perf/viewer/build.py]: tools/run_all_build.go

**Comment by arv** [comment]
LGTM

**Pull request merged**